### ISSUE TITLE
Version 4.20.0 has breaking changes to outputs

### ISSUE THREAD

**Issue by llamahunter** [terraform-aws-modules/terraform-aws-notify-slack#154]
## Description

All the outputs changed their names in version 4.20.0

## Versions

doesn't matter. verision 4.20.0 broke things.

## Reproduction

Try to use version 4.20.0 with existing terraform.

## Expected behavior
Breaking changes have major version number updates, as per semver.org

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner
```

## Actual behavior
The 'this_' prefix on all outputs were removed

**Comment by bryantbiggs** [comment]
see https://github.com/terraform-aws-modules/terraform-aws-notify-slack/pull/151#issuecomment-991388987

**Comment by llamahunter** [comment]
This is an unprofessional response (i.e. downstream users need to fix their code to track backwards incompatible changes in minor version number releases).  Please follow SemVer semantics https://semver.org/

```
Given a version number MAJOR.MINOR.PATCH, increment the:

MAJOR version when you make incompatible API changes,
MINOR version when you add functionality in a backwards compatible manner, and
PATCH version when you make backwards compatible bug fixes.
Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.
```

**Comment by antonbabenko** [comment]
This issue has been resolved in version 4.24.0 :tada:
